js/toggle.js

```diff
@@ -6,12 +6,13 @@ jQuery(document).ready(function() {
 	else
 		expand_default(selected);
 	jQuery(window).on('load',function () {
-		if(item_type == "none")
+		if(item_type == "none") {
 			item_hidden();
-		else if(selected != "1" && item_type != "none")
-			item_hidden();
-		else
+		} else if(selected != "1" && item_type != "none") {
+			// Remove duplicate call to item_hidden() here

```

Comment:
nothing is written in this condition

we can combine these 2 conditions - 

if(item_type == "none") {
			item_hidden();
		} else if(selected != "1" && item_type != "none") {
			item_hidden();
		}

Reply:
done @sushmak02